.JuliaFormatter.toml

```diff
@@ -0,0 +1,15 @@
+# Options for the JuliaFormatter auto syntax formatting tool.
+# https://domluna.github.io/JuliaFormatter.jl/stable/
+# https://docs.sciml.ai/SciMLStyle/stable/
+
+# We don't use SciML style since aligning with opening brackets doesn't look good if
+# the opening bracket is near the end of the line. This squeezes the code to the right,
+# and this alignment confuses auto indent.
```

Comment:
```suggestion
```

This is outdated. I just tried switching to SciML style, and this particular issue is fixed, though overall I still find in worse looking than the default style. I just removed these lines in https://github.com/Deltares/Ribasim/pull/1614 as well.

Reply:
Thanks, that is good to know.

In the past I have used Blue without any modifications.
For my private projects I prefer Blue over SciML because I think it looks better.
You might want to give it a go and see whether aesthetically it is a better fit for us.
It will for sure have my vote.

Reply:
I agree with Blue over SciML, but this is default, not SciML. What do you like better about adding `style = "blue"`? I just tried it out, the differences are not large, but most differences I dislike:

- Blue removes spaces around = in kwargs
- Blue tries harder to put more args on the same line

Both make Blue more compact but less readable IMO. The default style is also more in line with Python's black formatter.

Reply:
In addition we were already using the default style.

Reply:
Indeed, [BlueStyle](https://github.com/JuliaDiff/BlueStyle) definitely goes beyond the [Julia Style Guide](https://docs.julialang.org/en/v1/manual/style-guide/). I was only really thinking about the JuliaFormatter setting.